pkg/reconciler/v1alpha1/revision/resources/deploy_test.go

```diff
@@ -498,28 +464,15 @@ func TestMakePodSpec(t *testing.T) {
 		}),
 	}, {
 		name: "concurrency=multi, readinessprobe=http",
-		rev: &v1alpha1.Revision{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace: "foo",
-				Name:      "bar",
-				UID:       "1234",
-				Labels:    labels,
-			},
-			Spec: v1alpha1.RevisionSpec{
-				ContainerConcurrency: 0,
-				Container: corev1.Container{
-					Image: "busybox",
-					ReadinessProbe: &corev1.Probe{
-						Handler: corev1.Handler{
-							HTTPGet: &corev1.HTTPGetAction{
-								Path: "/",
-							},
-						},
+		rev: revision(func(revision *v1alpha1.Revision) {
+			revision.Spec.Container.ReadinessProbe = &corev1.Probe{
```

Comment:
Since this is reused, maybe add a `withReadinessProbe` helper, which only takes the `corev1.Handler`?

Reply:
Done.